standalone_tests/test_point_cloud_scene.py

```diff
@@ -0,0 +1,48 @@
+import wandb
+import numpy as np
+
+wandb.init()
+
+wandb.log(
+        {
+            "point_scene": wandb.Object3D(
+                {
+                    "type": "lidar/beta",
+                    "points": np.array([[0.4, 1, 1.3], [1, 1, 1], [1.2, 1, 1.2]]),
+                    "boxes": np.array(
+                        [
+                            {
+                                "corners": [
+                                    [0,0,0],
+                                    [0,1,0],
+                                    [0,0,1],
+                                    [1,0,0],
+                                    [1,1,0],
+                                    [0,1,1],
+                                    [1,0,1],
+                                    [1,1,1]
+                                ],
+                                "label": "Tree",
```

Comment:
It might make more sense to call this markdown, or at least have an optional markdown key for storing extra metadata like probability.

Reply:
Let's leave as is for now